stable/ghost/templates/NOTES.txt

```diff
@@ -31,9 +32,9 @@ host. To configure Ghost with the URL of your service:
 {{- if eq .Values.serviceType "ClusterIP" }}
 
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
-  echo Blog URL  :  http://127.0.0.1:2368/
-  echo Admin URL : http://127.0.0.1:2368/ghost
-  kubectl port-forward $POD_NAME 2368:2368
+  echo Blog URL  :  http://127.0.0.1{{ if .Values.ghostPort }}:{{ .Values.ghostPort  }}{{ end }}/
```

Comment:
Yes, as @prydonius pointed out, accessing the service using the cluster IP will probably only work with solutions such as minikube.